### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
- [x] Added utilities for animations.
- [x] Added a dedicated component for animations.
- [x] Added a dependency to this component for all animations already present in the code (now deported).

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1697

File: packages/ng/style/components/_sidepanel.scss [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-  --components-sidepanel-translate: 3rem;
+	--components-sidepanel-size-small: 40rem;
+	--components-sidepanel-size-medium: 55rem;
+	--components-sidepanel-size-large: 70rem;
+	--components-sidepanel-translate: 3rem;
 }
 
 .lu-popup-panel {
 	&.mod-sidepanel {
+		@include animation.sidepanel;
```
Comment (jeremie-lucca) [inline packages/ng/style/components/_sidepanel.scss:12]: Est-ce que ça a du sens de sortir une animation spécifique à un composant ?

File: packages/scss/src/components/progress/mods.scss [hunk 2]
```diff
@@ -1,17 +1,9 @@
...
-		100% {
-			left: 100%;
-			right: -35%;
-		}
-	}
+@use '../animation/exports' as animation;
 
+@mixin indeterminate {
 	.progress-bar {
+		@include animation.progressIndetereminate;
```
Comment (jeremie-lucca) [inline packages/scss/src/components/progress/mods.scss:5]: Same

File: packages/scss/src/components/toast/mods.scss [hunk 3]
```diff
@@ -54,15 +56,6 @@
 }
 
 @mixin circularGauge {
-	@keyframes stroke {
```
Comment (jeremie-lucca) [inline packages/scss/src/components/toast/mods.scss:57]: Same

**Review by jeremie-lucca — DISMISSED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Pull request merged**